.travis.yml

```diff
@@ -1,16 +1,24 @@
 osx_image: xcode7.3
 language: node_js
+
 os:
-- osx
+  - osx
+  - linux
+  
 cache:
   directories:
-  - node_modules
+    - node_modules
+    
+script:
+  - npm run update-dependencies
```

Comment:
I think it should be fine to just use the latest `electron-builder` and remove the `npm run update-dependencies`